symplyphysics/laws/thermodynamics/stefan_boltzmann_law.py

```diff
@@ -0,0 +1,31 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+# Description
+## The Stefan–Boltzmann law, also known as Stefan's law, states that the total energy radiated per
+## unit surface area of a black body in unit time (known variously as the black-body irradiance, energy flux density, radiant flux, or the emissive power), j*, is directly proportional to the fourth
+## power of the black body's thermodynamic temperature T (also called absolute temperature). 
+
+# Law: j* = sigma*T^4, where
+## j* is irradiance,
+## sigma is constant of proportionality,called the Stefan–Boltzmann constant,
+## T is temperature of a completely black body
+
+# Note
+## j* = epsilon*sigma*T^4, where ε is the integral absorption capacity of the body. For a completely black body ε = 1.
+
+irradiance = Symbol("irradiance", units.power/units.length**2)
+temperature = Symbol("temperature", units.temperature)
+
+law = Eq(irradiance, units.stefan * (temperature**4))
+
+def print_law() -> str:
+    return print_expression(law)
+
+@validate_input( temperature_=temperature)
+@validate_output(irradiance)
+def calculate_pressure(temperature_: Quantity) -> Quantity:
+    solved = solve(law, irradiance, dict=True)[0][irradiance]
+    result_expr = solved.subs({temperature: temperature_})
```

Comment:
With only one variable to substitute, you can use shorter subs version:
`subs(temperature, temperature_)`